packages/calcite-components/src/components/table-cell/table-cell.scss

```diff
@@ -11,11 +11,11 @@
   @apply contents;
 }
 
-:host([alignment="center"]) td {
+:host([alignment="center"]) td:not(.selection-cell):not(.number-cell) {
```

Comment:
nitpick: I don't love these selectors.

Every time we add another td this selector needs to grow. I know we have a ton of these in the codebase but IMO we should discuss moving this logic into JS and use a css class to represent what this should be styling.

Reply:
Agreed, added a class for this.